Updating as per PREX-2204 (#893)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/content-customizations/product-recommendations.mdx

### When product context is required

Whether a seed SKU or anchor price is required depends on the **recommendation unit type** and the **price filter operator**, not on each other:

| Unit type | Seed SKU | Anchor price |
| --- | --- | --- |
| **Anchor-based** (more-like-this, visual similarity, viewed-viewed, viewed-bought, bought-bought) | Required | Only for dynamic or relative price filter operators |
| **Context-free** (most-viewed, most-purchased, and similar popularity or behavioral types) | Not required | Not applicable |

A **static** price filter defines a fixed price range in the filter rule. It does not mean the unit can run without an anchor product. For example, a more-like-this unit with a static price filter returns no products when the seed SKU is missing, but returns results with the seed SKU alone—`currentprice` is not required.

### Storefront block configuration

How you configure the block depends on the page type and unit type:

> - **PDP pages**: Set only `recId`. The boilerplate supplies the seed SKU from ACDL product context. Anchor price is included automatically when a dynamic or relative price filter operator requires it.

Reviewer: Same nuance: "included automatically when a dynamic or relative price filter operator requires it" implies the block conditions price-sending on the operator, but on ACO/ACCS PDPs it sends the ACDL price unconditionally and the backend decides whether to use it. Suggest:

```suggestion
- **PDP pages**: Set only `recId`. The boilerplate supplies the seed SKU from ACDL product context. On Adobe Commerce Optimizer and Adobe Commerce as a Cloud Service it also sends the ACDL anchor price automatically; the backend applies it only when the unit's filter uses a dynamic or relative operator.
```

   </table>

   :::note
   PDP pages require only `recId`, as shown above.

   Other pages that need product context require `recId` and `currentsku`. For example:

   <table style="width: 100%; min-width: 470px; max-width: 100%; table-layout: fixed; border-collapse: collapse;">
   <tbody>
   <tr>
   <td colspan="2" style="text-align: center; padding: 0.75rem; border: 1px solid var(--sl-color-gray-5); background-color: var(--sl-color-gray-6); font-weight: 600;">product-recommendations</td>
   </tr>
   <tr>
   <td style="width: 50%; padding: 0.75rem; border: 1px solid var(--sl-color-gray-5);">recId</td>
   <td style="width: 50%; padding: 0.75rem; border: 1px solid var(--sl-color-gray-5);">2ed7ea09-d9bd-4691-91c3-2c0a6d4fd869</td>
   </tr>
   <tr>
   <td style="width: 50%; padding: 0.75rem; border: 1px solid var(--sl-color-gray-5);">currentsku</td>
   <td style="width: 50%; padding: 0.75rem; border: 1px solid var(--sl-color-gray-5);">crz-eco-tr-std-2017</td>
   </tr>
   </tbody>
   </table>

>    When the unit uses dynamic price filter operators on Adobe Commerce Optimizer or Adobe Commerce as a Cloud Service, also set `currentprice` on non-PDP pages:

Reviewer 2: ```suggestion
   When using dynamic price filter operators on Adobe Commerce Optimizer or Adobe Commerce as a Cloud Service, set `currentprice` on non-PDP pages:
```
